server/views/referrals/checkAnswers.njk

```diff
@@ -0,0 +1,103 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% from "./_taskListSection.njk" import taskListSection %}
+{% from "../partials/audienceTags.njk" import audienceTags %}
+
+{% extends "../partials/layout.njk" %}
+
+{% block personBanner %}
+    {% include "./_personBanner.njk" %}
+{% endblock personBanner %}
+
+{% block backLink %}
+    {{ govukBackLink({
+    text: "Back",
+    href: "#"
+  }) }}
+{% endblock backLink %}
+
+{% block content %}
+    <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <p class="govuk-body">Read through this page and check that the information provided for each answer is accurate. When you feel the application is complete, submit it.</p>
+
+            <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-6">
+
+            <h2 class="govuk-heading-m">Application summary</h2>
+
+            {{ govukSummaryList({
+                classes: 'govuk-summary-list--no-border',
+                rows: applicationSummaryListRows,
+                attributes: {
+                    "data-testid": "application-summary-list"
```

Comment:
Nice! I've added a card for us to do this for other stuff we target in integration tests